x-pack/plugins/reporting/server/lib/export_types_registry.ts

```diff
@@ -59,9 +51,10 @@ export class ExportTypesRegistry {
       }
       const foundResult: ExportTypeDefinition = value;
 
-      if (result) {
-        throw new Error('Found multiple items matching predicate.');
-      }
+      // PDF works if this is commented out
+      // if (result) {
```

Comment:
@tsullivan - when I comment this out PDF reporting works. Do you know why this if clause and error were added?